packaging/nuget/templates/librdkafka.redist.targets

```diff
@@ -1,10 +1,10 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemDefinitionGroup>
     <Link>
-      <AdditionalDependencies Condition="'$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x64\native\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalDependencies Condition="'$(Platform)' != 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x86\native\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories Condition="'$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x64\native;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
-      <AdditionalLibraryDirectories Condition="'$(Platform)' != 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x86\native;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalDependencies Condition="'$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\build\native\lib\win7\x64\win7-x64-Release\v120\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Unnecessary dir-backtracking, the lib folder is in the same directory as the target sfile:
```
     1459  2017-11-30 07:49   build/native/librdkafka.redist.targets
   125600  2017-11-30 07:49   build/native/include/librdkafka/rdkafka.h
    70853  2017-11-30 07:49   build/native/include/librdkafka/rdkafkacpp.h
    39848  2017-11-30 07:49   build/native/lib/win7/x64/win7-x64-Release/v120/librdkafka.lib
    39098  2017-11-30 07:49   build/native/lib/win7/x64/win7-x64-Release/v120/librdkafkacpp.lib
    40610  2017-11-30 07:49   build/native/lib/win7/x86/win7-x86-Release/v120/librdkafka.lib
    38312  2017-11-30 07:49   build/native/lib/win7/x86/win7-x86-Release/v120/librdkafkacpp.lib
```

Reply:
good point